Connectors/Actionable Messages - Module 1 - Style guide & QA refresh

andrewconnell opened the pull request:
This submission covers two primary updates:

- updated module to reflective guidance in the [Microsoft style guide](https://docs.microsoft.com/en-us/style-guide/welcome/)
- refreshed all content, labs, screenshots and demos to ensure they are current

dclaux commented on ConnectorActionableMsgs/01 Build a basic Connector/Demos/01-card-webhook/Readme.md:
```diff
@@ -1,44 +1,46 @@
+1. Select **Done**. Then close the Connector and Group settings menus.
 
-    > Note: The URL will be used several times in this Exercise, as will the clipboard. We recommend pasting the URL into Notepad or other application.
+## Explore MessageCard playground
```
We are now calling it "Card Playground" instead of "MessageCard Playground." Ideally it should be changed everywhere "MessageCard Playground" is mentioned.

pschaeflein replied:
The url (https://messagecardplayground.azurewebsites.net) seems to be the same, correct?

dclaux replied:
Correct

pschaeflein replied:
One last thing -- in the playground, if you send the card via email, the subject line of the email reads "MessageCard Playground Test Card"   (I just tested it minutes ago.)

dclaux replied:
Yes, known issue. Haven't had a chance to fix it yet.

andrewconnell replied:
But in that case, are we supposed to update the module to reflect something that's not accurate in prod?

dclaux replied:
Yes please. The documentation is already updated.

dclaux commented on ConnectorActionableMsgs/01 Build a basic Connector/Demos/01-card-webhook/Readme.md:
```diff
@@ -1,44 +1,46 @@
 
-1. In another browser tab or window, navigate to **https://messagecardplayground.azurewebsites.net**.
-1. The playground site provides for uploading a custom card definition or reviewing several samples. Use the **select a sample** dropdown to select a sample that closely matches your requirements. (The image in this lab uses the **Connectors reference example**, but that is not required.)
+    > Note: The image in this lab uses the **Connectors reference example**, but that is not required.
```
That sample has been removed from the playground since Build 2018 - could we update to use another of the available samples?

BobGerman commented:
1. The Connector Developer's Dashboard under "Register Your Connector" has changed and no longer matches the screen shot and instructions. Specifically, the entire "Enable this connector for" section has been removed. Instructions need an update as well.
2. The connector URL in Exercise 2 Step 12 has changed and the lab does not work. Julie and I figured out this morning that it's necessary to download the URL from the Connector Developer's Dashboard; it isn't the same as in the lab any more.
3. Very minor but ... in Excercise 1 Step 3, the gear is in the upper right corner of the screen not upper left

Still working thru these labs in prep for a workshop next week; I'll post any other issues I find.

Thanks!

andrewconnell commented:
Addressed comments & feedback from @davidchesnut & @BobGerman above in commit https://github.com/OfficeDev/TrainingContent/pull/498/commits/f4f3bb4ba63c59b143fbe71771377e665846db02

BobGerman commented:
Thanks @andrewconnell - the other labs went smoothly.

These labs are great BTW, thanks so much! I think they changed the URL and dev dashboard during the last week, since the updates. Keeping up with the changes is impossible!
